static/styles/zulip.css

```diff
@@ -25,23 +25,28 @@ $right_sidebar_width: 250px;
 body,
 html {
     width: 100%;
-    height: 100%;
-    overflow-x: hidden;
-    overflow-y: hidden;
-
     touch-action: manipulation;
 }
 
 #app-loading.loaded {
     display: none !important; /* We are now loaded, by definition. */
 }
 
+html {
+    overflow-x: hidden;
+    overflow-y: auto;
```

Comment:
We should use `overflow-y: scroll` otherwise it has weird scrollbar appear - disappear effect.

Reply:
Thanks! Tested this, there was a bug in narrow widths where the content shifted a bit on opening modals due to this scrollbar effect.
This fixes it.